test/assertions/to-contain.spec.js

```diff
@@ -1,5 +1,10 @@
 /*global expect*/
 describe('to contain assertion', function () {
+    var expect = unexpected.clone().use(typeof weknowhow === 'undefined' ?
+        require('unexpected-magicpen') :
+        weknowhow.unexpectedMagicPen
+    );
+
```

Comment:
We could consider exposing the required unexpectedMagicpen plugin as a global from common.js